src/helper-modules/dun-and-bradstreet/dun-and-bradstreet.service.ts

```diff
@@ -0,0 +1,56 @@
+import { HttpService } from '@nestjs/axios';
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { DunAndBradstreetConfig } from '@ukef/config/dun-and-bradstreet.config';
+import { DUN_AND_BRADSTREET } from '@ukef/constants';
+import { HttpClient } from '@ukef/modules/http/http.client';
+
+import { createWrapDunAndBradstreetHttpGetErrorCallback } from './wrap-dun-and-bradstreet-http-error-callback';
+
+@Injectable()
+export class DunAndBradstreetService {
+  private readonly httpClient: HttpClient;
+  private readonly encodedKey: string;
+
+  constructor(httpService: HttpService, configService: ConfigService) {
+    this.httpClient = new HttpClient(httpService);
+    const { key } = configService.get<DunAndBradstreetConfig>(DUN_AND_BRADSTREET.CONFIG.KEY);
+    this.encodedKey = key;
+  }
+
+  async getDunAndBradstreetNumberByRegistrationNumber(registrationNumber: string): Promise<string> {
+    const path = `/v1/match/cleanseMatch?countryISOAlpha2Code=GB&registrationNumber=${registrationNumber}`;
```

Comment:
`GB` constant exist if not mistaken, please load this value from constant.

Reply:
I've had a look but can't find this, `GB`'s not used for anything else in the codebase either so I don't think it's worth adding a constant just for this case.